app/services/AmendReturnAnswerComparisonService.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import cacheables.ReturnDisplayApiCacheable
+import models.UserAnswers
+import pages.amends._
+
+class AmendReturnAnswerComparisonService {
+
+  def hasMadeChangesOnAmend(userAnswers: UserAnswers): Boolean = {
+    userAnswers.get(ReturnDisplayApiCacheable) match {
+      case None => throw new Exception("Original return missing from user answers")
+
+      case Some(original) =>
+
+        val manufacturedHasChanged: Boolean = !userAnswers.get(AmendManufacturedPlasticPackagingPage)
```

Comment:
potential refactor here, as we have discussed, up to you how much though, do not want to loose the readabilty you have! ⭐

Reply:
leaving as is for readability 👍